src/main/java/org/folio/circulation/domain/FeeFineRepository.java

```diff
@@ -20,16 +20,12 @@ public FeeFineRepository(Clients clients) {
     feeFineStorageClient = clients.feeFineStorageClient();
   }
 
-  public CompletableFuture<Result<FeeFine>> getOverdueFine(String feeFineOwnerId) {
-    return getFeeFine(feeFineOwnerId, FeeFine.OVERDUE_FINE_TYPE);
-  }
-
-  private CompletableFuture<Result<FeeFine>> getFeeFine(String feeFineOwnerId, String feeFineType) {
-    Result<CqlQuery> typeQuery = CqlQuery.exactMatch("feeFineType", feeFineType);
-    Result<CqlQuery> ownerQuery = CqlQuery.exactMatch("ownerId", feeFineOwnerId);
+  public CompletableFuture<Result<FeeFine>> getFeeFine(String type, boolean automatic) {
+    Result<CqlQuery> typeQuery = CqlQuery.exactMatch("feeFineType", type);
+    Result<CqlQuery> automaticQuery = CqlQuery.exactMatch("automatic", Boolean.toString(automatic));
```

Comment:
What does automatic mean for a fee fine type?

Does this mean that if the correct fee fine type is present, however not set to automatic, then no fee fine type will be found?

Reply:
`automatic` means that fee/fine was created automatically, i.e. without user's interaction. For example, an overdue fine can be created automatically in scope of a check-in process, according to some set of rules (e.g. overdue fine policy). These types of fees/fines are few, they are created from a predefined data set, and they will not be visible in the UI.

On the other hand, manual fee/fine types are created by a user manually. They have arbitrary names and can be edited by the user in the UI.

Reply:
Ok. The assumption is that the appropriate fee / fine type will be pre-defined.